### DISCUSSION TITLE
Select Options for Associate Actions

### DISCUSSION THREAD

**Discussion by KWangechi** [filamentphp/filament#13532] [Help]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

Hello everyone,

How can one define select option labels while creating an Associate Action while using Relational Managers; I have sth like that:

Tables\Actions\AssociateAction::make()
                    ->recordSelectOptionsQuery(function ($query) {
                        return $query->select('id', 'parcel_no_uniq', 'parcel_size');
                    })
                ->preloadRecordSelect()
                
Maybe have sth like what the Select class has for `Select::make('my_field')->getOptionLabelFromRecordUsing()`

**Comment by bretto36** [comment]
Found this searching for a solution to a different problem but thought i'd add my answer

`->recordTitle(fn (Class $record): string => "{$record->description} - ({$record->number})")`
